scripts/scil_tractogram_resample.py

```diff
@@ -56,29 +63,33 @@ def _build_arg_parser():
 
     p.add_argument('--never_upsample', action='store_true',
                    help='Make sure to never upsample a tractogram.\n'
-                        'Useful when downsample batch of files using bash.')
+                        'Useful when downsampling a batch of files using '
+                        'bash.')
 
     # For upsampling:
-    upsampling_group = p.add_argument_group('Upsampling params')
-    upsampling_group.add_argument('--point_wise_std', type=float, default=1,
-                                  help='Noise to add to existing streamlines '
-                                       'points to generate new ones [%(default)s].')
-    upsampling_group.add_argument('--tube_radius', type=float, default=1,
-                                  help='Maximum distance to generate streamlines '
-                                       'around the original ones [%(default)s].')
-    upsampling_group.add_argument('--gaussian', metavar='SIGMA', type=int,
-                                  help='Sigma for smoothing. Use the value of '
-                                       'surrounding X,Y,Z points on the '
-                                       'streamline to blur the streamlines.\n'
-                                       'A good sigma choice would be around 5.')
-    upsampling_group.add_argument('-e', dest='error_rate', type=float, default=0.1,
-                                  help='Maximum compression distance in mm '
-                                       '[%(default)s].')
-
-    upsampling_group.add_argument('--keep_invalid_streamlines',
-                                  action='store_true',
-                                  help='Keep invalid newly generated streamlines'
-                                       ' that may go out of the \nbounding box.')
+    up_group = p.add_argument_group('Upsampling params')
+    up_group.add_argument('--point_wise_std',
+                          type=ranged_type(float, 0, None, min_excluded=True),
+                          default=1,
+                          help='Noise to add to existing streamlines '
+                               'points to generate new ones [%(default)s].')
+    up_group.add_argument('--tube_radius',
+                          type=ranged_type(float, 0, None, min_excluded=True),
+                          default=1,
+                          help='Maximum distance to generate streamlines '
+                               'around the original ones [%(default)s].')
```

Comment:
I would add a unit there. What is "1" here ? 1 voxel ? 1mm ?

Reply:
I wrote mm. That's right, @frheault ?

Reply:
not sure I don't see any formula with the voxel resolution. Once we know we should update it to each function it uses it.

Reply:
The operation is performed in word space (rasmm) so yes, it would be In mm